src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -263,7 +263,12 @@ export default compose(
         },
         workspaceInviteMessageDraft: {
             key: ({route}) => `${ONYXKEYS.COLLECTION.WORKSPACE_INVITE_MESSAGE_DRAFT}${route.params.policyID.toString()}`,
-            selector: (draft) => (isEmpty(draft) ? '' : draft),
+            selector: (draft) => {
+                if (_.isUndefined(draft)) {
```

Comment:
@DylanDylann 
I think @bernhardoj is correct https://github.com/Expensify/App/issues/32835#issuecomment-1862254278 too.

and less code is better. so let's remove the selector to fix the issue ? what do you think?

Reply:
@rushatgabhane I just updated. Please help check